weblate/settings_openshift.py

```diff
@@ -94,6 +94,29 @@
     }
 }
 
+if 'REDIS_PASSWORD' in os.environ:
+    CELERY_TASK_ALWAYS_EAGER = False
+    CELERY_BROKER_URL = '"redis://:{0}@{1}:{2}"'.format(
+          os.environ.get('REDIS_PASSWORD'),
+          os.environ.get('REDIS_HOST'),
+          os.environ.get('REDIS_PORT')
+      )
+    CELERY_RESULT_BACKEND = CELERY_BROKER_URL
+
+    CELERY_WORKER_PREFETCH_MULTIPLIER = 0
+    CELERY_WORKER_MAX_MEMORY_PER_CHILD = 200000
+    CELERY_BEAT_SCHEDULE_FILENAME = os.path.join(
+        DATA_DIR, 'celery', 'beat-schedule'
+    )
+    CELERY_TASK_ROUTES = {
+        'weblate.trans.search.*': {'queue': 'search'},
+        'weblate.trans.tasks.optimize_fulltext': {'queue': 'search'},
+        'weblate.trans.tasks.cleanup_fulltext': {'queue': 'search'},
+        'weblate.memory.tasks.*': {'queue': 'memory'},
+        'weblate.accounts.tasks.notify_change': {'queue': 'notify'},
+    }
```

Comment:
You already get the above 4 settings from settings_example, so better not to copy them.

Reply:
Right, I've removed this. Thanks.